src/components/generic/InputField/InputField.scss

```diff
@@ -99,8 +105,9 @@ $C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-input-field;
     right: 0;
     transform: translateY(-50%);
 
-    &:not(:last-child) {
-      right: 2rem;
+    right: 2rem;
```

Comment:
i would prefer to keep the previous syntax, which means "put a margin if it's not the last child", instead of "put a margin, but if it's the last child, remove it"

of course, updated to "second-to-last"

Reply:
and you can actually use the simpler `:nth-last-child(n+3)`, which means any child that matches any index between 3 and infinity (n goes from 0 to infinity)